dune/grid/cpgrid/Iterators.hpp

```diff
@@ -113,6 +113,32 @@ namespace Dune
     } // namespace cpgrid
 } // namespace Dune
 
+namespace std
+{
+  template< int codim, Dune::PartitionIteratorType pitype >
```

Comment:
Sorry for being annoying, but I'd like two formatting changes:
 - Use 4-indentation since that is what the rest of the file is like.
 - Use `typedef value_type* pointer`, the asterisk should be considered part of the pointer type, same goes for the reference `&`, in both places below.